concordium-std/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased changes
+- Handle new cases for NewContractNameError and NewReceiveNameError
```

Comment:
This needs to be a bit clearer. As it is it really doesn't say anything to the users.